app/services/vot/parser.rb

```diff
@@ -4,37 +4,71 @@ module Vot
   class Parser
     class ParseException < StandardError; end
 
-    Result = Data.define(
+    class Result < Data.define(
+      :type,
       :component_values,
+      :component_separator,
+      :asserted_ial_acr,
       :aal2?,
       :phishing_resistant?,
       :hspd12?,
       :identity_proofing?,
       :biometric_comparison?,
+      :biometric_comparison_required?,
```

Comment:
When I am using this class downstream to determine if I should perform biometric comparison what am I to make of `biometric_comparison?` and `biometric_comparison_required?`? Which one should I inspect to determine if I should perform biometric comparison or both? Is there something in the design of this that I am missing that should signal the answer to that question?

Reply:
The idea is that if the SP wants to require a user to complete proofing with biometric comparison (instead of preferring it when the user has already completed biometric comparison), this would be used as opposed to testing the asserted_ial_acr value directly. I can revert this as it seems to confuse the issue.